Dockerfile-py2.7

```diff
@@ -25,11 +25,14 @@ ENV DISPLAY :99
 
 WORKDIR /workspace/Qt.py
 ENTRYPOINT cp -r /Qt.py /workspace && \
-    python build_caveats_tests.py && \
+    python build_caveats.py && \
+    python build_membership.py && \
 	Xvfb :99 -screen 0 1024x768x16 2>/dev/null & \
-	sleep 3 && \
+    while ! ps aux | grep -q '[0]:00 Xvfb :99 -screen 0 1024x768x16'; do echo "Waiting for Xvfb..."; sleep 1; done && \
```

Comment:
The new command just checks whether `grep` finds this string:

    0:00 Xvfb :99 -screen 0 1024x768x16

The original command which launches Xvfb is untouched. So the functionality should be unchanged.